students/ChelseaSmith/Lesson4/mailroom.py

```diff
@@ -0,0 +1,84 @@
+# define variables
+record = {"James Smith": [85, 100, 75],  # dictionary with key as donor name and value as list of previous donations
+          "Alex Jones": [25, 25, 25],
+          "Kate Smith": [100],
+          "Kate Jones": [175, 50],
+          "Alex Smith": [200, 200, 200]}
+
+choice = 1  # variable used to choose main menu options
+
+# functions
+
+
+def report(dictin):  # this prints a report given database dictin
+    print(f"Donor name" + " " * 17 + " | Total Given | Num Gifts | Average Gift\n")
+    for k in dictin:
+        sum = 0
+        num = 0
+        for item in dictin[k]:
+            sum = sum + int(item)
+            num = num + 1
```

Comment:
couple comments here:
* if you're looping over dict and you intend to use both keys and values like here then it is more pythonic to use `.items()` -> `for donor_name, donations in dictin.items()` this is done to improve readability.
* you don't need to loop over list to get the data you need. see example below.

```
for name, donations in records.items():
    total = sum(donations)
    num_of_donations = len(donations)
    avg = total / num_of_donations
```

above is readable and uses built-in functions to get your stats.